include/dpp/channel.h

```diff
@@ -409,7 +409,6 @@ class DPP_EXPORT channel : public managed, public json_interface<channel>  {
 	 * Does not return reliable information for voice channels, use
 	 * dpp::channel::get_voice_members() instead for this.
 	 * @return A map of guild members keyed by user id.
-	 * @note If the guild this channel belongs to is not in the cache, the function will always return 0.
```

Comment:
guilds are always cached, so this note is kinda useless isn't it?

Reply:
no this isn't useless, it may be called before the on_guild_create for the guild, if you give it a channel object you've constructed yourself with a hard coded id, or if the guild is unavailable when the bot starts due to an outage, even a threading race condition could cause it if you attach your own threads to on_guild_create with a worker pool.
worth keeping as its an outcome that can happen non the less.

Reply:
gotcha, changed back